aws_cloudtrail_rules/aws_root_activity.py

```diff
@@ -15,19 +15,14 @@ def rule(event):
 
 
 def dedup(event):
-    return (
-        event.get("sourceIPAddress", "<UNKNOWN_IP>")
-        + ":"
-        + lookup_aws_account_name(event.get("recipientAccountId"))
-    )
+    return event.get("eventName") + ":" + lookup_aws_account_name(event.get("recipientAccountId"))
```

Comment:
I think dedup'ing on eventNames could potentially make this quite a noisy alert.  With the source ip address, we are still alerted on root activity, but the activity could be grouped together